prime-router/src/main/kotlin/transport/SftpTransport.kt

```diff
@@ -35,7 +35,9 @@ class SftpTransport : ITransport {
             val (user, pass) = lookupCredentials(receiver.fullName)
             // Dev note:  db table requires body_url to be unique, but not external_name
             val fileName = Report.formExternalFilename(header)
-            uploadFile(host, port, user, pass, sftpTransportType.filePath, fileName, header.content)
+            val sshClient = connect(host, port, user, pass)
```

Comment:
This was the biggest change to SftpTransport - the uploadFile call was doing both a connect and an upload.   I separated these, so I could call connect, and an 'ls', instead of upload.  

This also makes it easy to see cases where the connection works, but the file upload or 'ls' does not work.